--- /usr/share/bluebuild/justfiles/00-default.just	1970-01-01 01:00:00.000000000 +0100
+++ /usr/share/bluebuild/justfiles/00-default.just	2026-03-31 17:29:56.355324632 +0100
@@ -84,7 +84,7 @@

 alias upgrade := update

-# Update system, flatpaks, and containers all at once
+# Update system, flatpaks, and uv tools all at once
 update VERB_LEVEL="full":
     #!/usr/bin/bash
     if [ {{ VERB_LEVEL }} = "help" ]; then
@@ -143,6 +143,9 @@

     # Run flatpak update
     update_command "flatpak update -y"
+
+    # Run uv tool upgrade
+    update_command "uv tool upgrade --all --refresh"
+
+    # Run distrobox upgrade
+    update_command "distrobox upgrade --all"

 alias changelog := changelogs
